MerlinAU.sh

```diff
@@ -3635,24 +3664,50 @@ _ChangelogVerificationCheck_()
     fi
 }
 
-##------------------------------------------##
-## Modified by ExtremeFiretop [2024-May-25] ##
-##------------------------------------------##
+##----------------------------------------##
+## Modified by Martinski W. [2024-May-31] ##
+##----------------------------------------##
 _ManageChangelog_()
 {
-    local mode="$1"  # Mode should be 'download' or 'view'
-    local wgetLogFile changeLogTag changeLogFile changeLogURL
+    if [ $# -eq 0 ] || [ -z "$1" ]
+    then echo "**ERROR** **NO_PARAMS**" ; return 1 ; fi
+
+    local mode="$1"  # Mode should be 'download' or 'view' #
+    local newUpdateVerStr=""
+    local wgetLogFile  changeLogFile  changeLogTag  changeLogURL
 
     # Create directory to download changelog if missing
     if ! _CreateDirectory_ "$FW_BIN_DIR" ; then return 1 ; fi
 
-    changeLogTag="$(echo "$(nvram get buildno)" | grep -qE "^386[.]" && echo "386" || echo "NG")"
-    if [ "$changeLogTag" = "386" ]
+    if [ "$mode" = "view" ]
     then
-        changeLogURL="${CL_URL_386}"
-    elif [ "$changeLogTag" = "NG" ]
+        if [ "$fwInstalledBaseVers" -eq 3006 ]
```

Comment:
Again wondering if this is a situation were we should be hard coding 3006 or if we should say anything higher than 3004

Reply:
Actually, because this is for changelogtags specifically, this is fine to hard code, I highly doubt that changelogs for version 3008 would be called changelog-3006 :) But the nvram calls above may still be valid in newer versions.